src/TypeScript/DefinitionWriter.cpp

```diff
@@ -5,6 +5,8 @@
 #include <clang/AST/DeclObjC.h>
 #include <iterator>
 
+bool apply_manual_changes = false;
```

Comment:
You'd better make this variable a public static field of the `DefinitionWriter` class instead of global and rename it to conform to other variables' camelCase convention.